R/render.R

```diff
@@ -69,41 +64,66 @@ create_prevelance_renderer <- function(
 
 #' @title Render incidence statistics
 #' 
-#' @description renders incidence (new for this timestep) for indivduals
+#' @description renders incidence (new for this timestep) for individuals
 #' 
 #' @param birth variable for birth of the individual
 #' @param renderer object for model outputs
 #' @param target incidence population
-#' @param source_pop the population which is sampled for infection
-#' @param prob probability of infection
 #' @param prefix for model outputs
 #' @param lowers age bounds
 #' @param uppers age bounds
 #' @param timestep current target
 #' 
 #' @noRd
 incidence_renderer <- function(
-  birth,
-  renderer,
-  target,
-  source_pop,
-  prob,
-  prefix,
-  lowers,
-  uppers,
-  timestep
-  ) {
+    birth,
+    renderer,
+    target,
+    prefix,
+    lowers,
+    uppers,
+    timestep
+) {
   for (i in seq_along(lowers)) {
     lower <- lowers[[i]]
     upper <- uppers[[i]]
     in_age <- in_age_range(birth, timestep, lower, upper)
-    renderer$render(paste0('n_', lower, '_', upper), in_age$size(), timestep)
     renderer$render(
       paste0('n_', prefix, lower, '_', upper),
       in_age$copy()$and(target)$size(),
       timestep
     )
+  }
+}
 
+#' @title Render probability of incidence statistics
+#' 
+#' @description renders probability of incidence (new for this timestep) for individuals
+#' 
+#' @param birth variable for birth of the individual
+#' @param renderer object for model outputs
+#' @param source_pop the population which is sampled for infection (bitten SAU for incidence, infecte for clinical/severe)
+#' @param prob probability of infection
+#' @param prefix for model outputs
+#' @param lowers age bounds
+#' @param uppers age bounds
+#' @param timestep current target
+#' 
+#' @noRd
+incidence_probability_renderer <- function(
```

Comment:
Whenever I look at this it always takes me a few to work out what it even means. I'd be happy to remove it, but agree it would be good to check if anyone uses it.